doc/source/tutorial/optimize.rst

```diff
@@ -740,6 +739,198 @@ And the optimization problem is solved with:
+     - ✓
+     - ✓
+   * - Equality constraints
+     -
+     -
+     -
+     -
+     -
+     -
+     - ✓
+   * - Inequality constraints
+     -
+     -
+     - ✓
+     -
+     -
+     -
+     - ✓
+
+
+.. list-table:: Optimization methods from :func:`minimize`
+   :widths: 5 5 5 5 5 5 5
+   :header-rows: 1
+
+   * -
+     - ``TNC``
+     - ``Newton-CG``
+     - ``dogleg``
+     - ``trust-ncg``
+     - ``trust-exact``
+     - ``trust-Krylov``
+   * - First derivatives
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+   * - Second derivatives
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+   * - Iterative Hessian factorization
+     - ✓
+     - ✓
+     -
+     - ✓
+     -
+     - ✓
+   * - Convergence rate
+     - S*
+     - S*
+     - Q
+     - S*
+     - Q
+     - S*
+   * - Convergence to stationary point
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+   * - Trust region
+     - LS
+     - LS
+     - TR
+     - TR
+     - TR
+     - TR
+   * - Bound constraints
+     - ✓
+     - ✓
+     -
+     -
+     -
+     -
+   * - Equality constraints
+     -
+     -
+     -
+     -
+     -
+     -
+   * - Inequality constraints
+     -
+     -
+     -
+     -
+     -
+     -
+
+.. list-table:: Optimization methods from :func:`minimize`
+   :widths: 5 35
+   :header-rows: 1
+
+   * -
+     - Description
+   * - Version added
+     - Specifies the algorithm’s first appearance in SciPy. Algorithms with version added 0.6*' were added in version 0.6 or before.
+   * - Wrapper
+     - Indicates whether the implementation available in SciPy wraps a function written in a compiled language (for example, C or FORTRAN)
+   * - First derivatives and Second derivatives
+     - Indicate whether first or second order derivatives are required. When ‘second derivatives’ is flagged with ‘~’, the algorithm accepts but does not require second-order derivatives from the user; it computes an approximation internally and uses it to accelerate method convergence.
+   * - Iterative Hessian factorization
+     - Denotes algorithms that factorize the Hessian in an iterative way, which does not require explicit matrix factorization or storage of the Hessian.
+   * - Local convergence
+     - Gives a lower bound on the rate of convergence of the iteration sequence once the iterate is sufficiently close to the solution: linear (L), superlinear (S) and quadratic (Q). Convergence rates denoted S* indicate that the algorithm has a superlinear rate for the parameters used in SciPy, but can achieve a quadratic convergence rate with other parameter choices.
+   * - Global convergence
+     - is marked for the algorithms with guarantees of convergence to a stationary point (that is, a point x* for which :math:`\nabla f \left(x^* \right) = 0`); this is not a guarantee of convergence to a global minimum.
+   * - Trust region
+     - Indicates which of the two globalization approaches is used by the algorithm, 'lines-search'(LS) or 'trust-region' (TR).
```

Comment:
```suggestion
     - Indicates which of two globalization approaches is used by the algorithm, 'line-search' (LS) or 'trust-region' (TR).
```
"the two globalization approaches" suggests that there are only two "globalization approaches". Presumably there are others, since Nelder-Mead uses neither?